src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -186,7 +188,7 @@ function MoneyRequestView({
 
     const currency = policy ? policy.outputCurrency : PolicyUtils.getPersonalPolicy()?.outputCurrency ?? CONST.CURRENCY.USD;
 
-    const mileageRate = TransactionUtils.isCustomUnitRateIDForP2P(transaction) ? DistanceRequestUtils.getRateForP2P(currency) : rates[rateID as string] ?? {};
+    const mileageRate = TransactionUtils.isCustomUnitRateIDForP2P(transaction) ? DistanceRequestUtils.getRateForP2P(currency) : distanceRates[rateID as string] ?? {};
```

Comment:
Please resolve [this consideration](https://github.com/Expensify/App/pull/38543#discussion_r1572686251) about when the mileage rate mapping has changed, and also add some manual test steps for it.